core/modules/layout/plugins/context/layout_context.inc

```diff
@@ -228,6 +283,7 @@ class LayoutContextBroken extends LayoutContext {
  * Pass-through context used when an unknown argument is encountered.
  */
 class LayoutOverridesPathContext extends LayoutContext {
+  var $usageType = LayoutContext::SYSTEM;
```

Comment:
Shouldn't this be `USAGE_TYPE_SYSTEM` instead?:

```suggestion
  var $usageType = LayoutContext::USAGE_TYPE_SYSTEM;
```

Reply:
Thanks! I renamed the constants several times and didn't catch this one.